strawberryfields/engine.py

```diff
@@ -609,22 +609,20 @@ def connection(self) -> xcc.Connection:
         )
 
     @property
-    def device_spec(self) -> DeviceSpec:
+    def device(self) -> Device:
         """The specification of the target device.
```

Comment:
I must admit, the fact that the docstring summary doesn't change makes me wonder if the renaming is necessary?